lib/data/replacements.json

```diff
@@ -206,6 +207,18 @@
       }
     ]
   },
+  "docker-compose": {
+    "description": "Compose has been integrated in the official Docker image.",
+    "packageRules": [
+      {
+        "matchDatasources": ["docker"],
+        "matchPackageNames": ["docker/compose"],
+        "matchCurrentValue": "/^((debian|alpine)-)?1\\.29\\.2$/",
+        "replacementName": "docker",
+        "replacementVersion": "27.1.1-cli"
```

Comment:
is this the lowest version which includes compose?

Reply:
It has been included in `20`, but I didn't find a tag like `20-cli` or `20.0.0` or `20.0.0-cli`. `27.1.1-cli` is IMO better because it's a version with major.minor.patch and only includes the cli binaries.

`23.0.0-cli` is lowest version in this format. Should I change the migration to that version?

Reply:
yes

Reply:
```suggestion
        "replacementVersion": "23.0.0-cli"
```